### DISCUSSION TITLE
FAILED TO CONNECT TO SPICE

### DISCUSSION THREAD

**Discussion by Davide002001** [utmapp/UTM#6362] [UTM Help (iOS)]
![IMG_0509](https://github.com/utmapp/UTM/assets/74309910/1a18a21e-1ee8-484c-8e79-463656b00194)

Hi, when I run a virtual machine on an iPad, but also on iPhone it gives me this error : Failed to Connect to SPICE: connection error to SPICE. how can I resolve this problem? I’m running a virtual machine with Linux mint.

**Comment by jeffwlin** [comment]
I resolved the issue by changing the "Sharing" setting to "VirtFS" instead of "SPICE WebDAV." It is under "Edit"->"Sharing" Hope that helps.

**Comment by lgrachov** [comment]
Same! I also have the same issue and changing the `Sharing` setting to `VirtFS` did not work and I still have issues with VM startup. My virtual machine is MacOS 9.2.1.

**Reply by bill-raynor** [reply]
Also Same. I am also using `VirtFS` on MacOS 14.6.1, UTM Version 4.5.4 (100),  UTM Remote 4.5.3 (fresh install from App store) on a Ubuntu 22.04 LTS. I have installed all the tools mentioned under Guest Support->Linux

**Comment by aplsms** [comment]
I have the same issue.

**Comment by skapsarali** [comment]
i have the same issue after install win 11 , install spice tool then its saying Display "Output Not Active"

**Comment by lefarris** [comment]
After hours of struggling, i resolved it finally by enabling automatic portforwarding through UPNP in my router. It seems that SPICE is opening dynamic ports when connecting over internet.

**Comment by ThiagoMarques** [comment]
I had a same issue but im change sharing to "VirtFS" instead of "SPICE WebDAV.

Mac OS Sequoia 16.4 / Win 11 on UTM app

**Comment by steveof2620** [comment]
I also have this issue. Out of nowhere it stopped working. No changes were made. Tried setting sharing to "VirtFS" instead of "SPICE WebDAV, did not work. Really need to get this up and running again.
